tds-item-selection-components/src/main/java/tds/itemselection/selectors/impl/AdaptiveSelector2013.java

```diff
@@ -41,11 +41,7 @@
   private static Logger logger = LoggerFactory.getLogger(AdaptiveSelector2013.class);
 
   // a combination of test-constant and examinee-variable data
-  private Blueprint blueprint;
-  private Cset1 cset1;
-  private ItemCandidatesData itemCandidates;
-  private TestSegment segment;
-  private Random rand = new Random();
```

Comment:
This was a port of the legacy code.  I'm a little surprised it never happened prior releases.  There is no reason for these Objects to be stored at a class level.  They are all created via the underlying methods or passed into the public method. 

The code was relying on instance level properties rather than passing the required objects into the private methods.  Since this selector is a singleton we were running into issues where the adaptive selector was modifying the collection in one thread while another thread was trying to work with it.